src/main/java/com/faforever/moderatorclient/ui/main_window/BracketRatingController.java

```diff
@@ -2,6 +2,7 @@
 
 import com.faforever.moderatorclient.ui.Controller;
 import javafx.fxml.FXML;
+import javafx.scene.control.Button;
```

Comment:
You gotta optimize the imports. Since there is no other change to this file it must be unused.